src/test/scala/ai/privado/languageEngine/python/PythonS3TaggerTest.scala

```diff
@@ -95,6 +77,12 @@ class PythonS3TaggerTest extends AnyWordSpec with Matchers with BeforeAndAfterAl
         |
         |bucket = get_s3().Bucket(OPTIMIZELY_EVENTS_BUCKET)
         |""".stripMargin)
+      .withRuleCache(ruleCache)
+
+    val databaseDetailsCache = DatabaseDetailsCache()
+    new RegularSinkTagger(cpg, ruleCache, databaseDetailsCache).createAndApply()
+    new PythonDBConfigTagger(cpg, databaseDetailsCache).createAndApply()
+    new PythonS3Tagger(cpg, s3DatabaseDetailsCache, databaseDetailsCache).createAndApply()
```

Comment:
This is not needed, as the below test case, need to be ported to check on the output generated